### PR TITLE
Bug#1512813 - added notes for NO_PROXY variable for scaleup playbook

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1512813

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@nekop Can you please check this one?

**Comment by nekop** [comment]
@gaurav-nelson LGTM

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation PTAL

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 1]
```diff
@@ -113,6 +113,11 @@ the configuration playbooks on the new hosts only. Before running the
 xref:../install_config/install/host_preparation.adoc#install-config-install-host-preparation[host
 preparation] steps.
 
+[IMPORTANT]
+====
+Scaleup playbook only configures the new host. It does not update *_NO_PROXY_* in master services and it does not restart master services.
```
Comment (ahardin-rh) [inline install_config/adding_hosts_to_existing_cluster.adoc]: The scaleup playbook

File: install_config/http_proxies.adoc [hunk 2]
```diff
@@ -96,6 +96,11 @@ all hosts, and effectively disables the proxy.
 Each name in this list is matched as either a domain which contains the host
 name as a suffix, or the host name itself.
 
+[NOTE]
+====
+When scaling up nodes, it is recommended to use a domain name rather than a list of hostnames.
```
Comment (ahardin-rh) [inline install_config/http_proxies.adoc]: s/it is recommended to use a/use a

**Comment by ahardin-rh** [comment]
@gaurav-nelson Just small nits from me. Otherwise, looks good!

**Comment by gaurav-nelson** [comment]
Thank you @ahardin-rh 
Updated, rebased, and ready to merge.

**Pull request merged**